client/my-sites/guided-transfer/host-credentials-page/index.jsx

```diff
@@ -0,0 +1,67 @@
+/**
+ * External dependencies
+ */
+import React, { PropTypes, Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Bluehost from './bluehost';
+import SiteGround from './siteground';
+import SectionHeader from 'components/section-header';
+
+class HostCredentialsPage extends Component {
+	constructor() {
+		super();
+		this.state = { fieldValues: {} };
```

Comment:
I'm not 100% sure about using the store vs state here but I think its worth some discussion